spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe "Contestants Index Page" do
+  let!(:recycled_material_challenge) { Challenge.create(theme: "Recycled Material", project_budget: 1000) }
+
+  let!(:news_chic)  { recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper") }
+  let!(:boardfit) { recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes") }
+
+  let!(:jay) { Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13) }
+  let!(:gretchen) { Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12) }
+  let!(:erin) { Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15) }
+
+  let!(:contestant_project_1) { ContestantProject.create(contestant_id: jay.id, project_id: news_chic.id) }
+  let!(:contestant_project_2) { ContestantProject.create(contestant_id: gretchen.id, project_id: news_chic.id) }
+  let!(:contestant_project_3) { ContestantProject.create(contestant_id: gretchen.id, project_id: boardfit.id) }
+
+  describe "As a visitor" do
+    it "shows a list of contestants' names and under each name there is a list of projects' names that they've been on" do
+      visit "/contestants"
+
+      within "contestant-#{jay.id}"
+      expect(page).to have_content(jay.name)
+      expect(page).to have_content(news_chic.name)
+
+      within "contestant-#{gretchen.id}"
+      expect(page).to have_content(gretchen.name)
+      expect(page).to have_content(news_chic.name)
+      expect(page).to have_content(boardfit.name)
+    end
+  end
```

Comment:
Great job making use of `within` blocks. I'm not sure if this is some more advanced use case of the `within` block without the accompanying do/end but I would recommend adding them in to avoid any tests not behaving as intended.

The only suggestion I would make is to make sure that your assertions match the expectation in the user story. If the user story states that the projects should show up in a list, I would make sure that it showed up as a list in my tests as well.

```ruby
expect(page).to have_content(["News Chic"])

expect(page).to have_content(["News Chic", "Boardfit"])
```